website/blog/2018-02-26-1.11.0.md

```diff
@@ -551,6 +551,7 @@ In Prettier 1.10.2, we had a lot of logic to handle formatting of Microsoft-brow
 #### Don't lowercase nesting selector ([#4048] by [@evilebottnawi])
 
 In Prettier 1.10.2, we were incorrectly lower-casing nesting SCSS selectors:
+
```

Comment:
These line breaks aren't actually new in 1.11.0, I just edited this via the GitHub editor, so it wasn't prettified.